packages/web-components/src/kumo/link/link.stories.ts

```diff
@@ -0,0 +1,71 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../../helpers.stories.js';
+import type { Link as KumoLink } from '../../link/link.js';
+import { LinkAppearance } from '../../link/link.options.js';
+
+type LinkStoryArgs = Args & KumoLink;
+type LinkStoryMeta = Meta<LinkStoryArgs>;
+
+const storyTemplate = html<LinkStoryArgs>`
+  <kumo-link href="${x => x.href}" appearance="${x => x.appearance}">${x => x.content}</kumo-link>
+`;
+
+export default {
+  title: 'Components/Kumo/Link',
+  args: {
+    content: 'Link',
+    href: '#',
+    disabled: false,
+    disabledFocusable: false,
+  },
+  argTypes: {
+    content: {
+      control: 'Anchor text',
+    },
+    appearance: {
```

Comment:
Still supporting link appearance in kumo?

Reply:
Yeah. There's Subtle (and Over-Brand 🤔) currently in the comps.